pkg/apis/apm/v1/webhook_test.go

```diff
@@ -97,20 +97,20 @@ func TestWebhook(t *testing.T) {
 			Operation: admissionv1beta1.Create,
 			Object: func(t *testing.T, uid string) []byte {
 				apm := mkApmServer(uid)
-				apm.Spec.Version = "7.2.0"
+				apm.Spec.Version = "7.4.0"
 				apm.Spec.KibanaRef = commonv1.ObjectSelector{Name: "kbname", Namespace: "kbns"}
 				return serialize(t, apm)
 			},
 			Check: test.ValidationWebhookFailed(
-				`spec.kibanaRef: Forbidden: required version for Kibana association is 7.3.0 but desired version is 7.2.0`,
+				`spec.kibanaRef: Forbidden: required version for Kibana association is 7.5.1 but desired version is 7.4.0`,
```

Comment:
Late nit: should read "minimum required version..."

Reply:
Thanks for the feedback, I'll fix this as part of https://github.com/elastic/cloud-on-k8s/issues/3154